### PR TITLE
[Outlook] (autolaunch) Add Windows support

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

File: docs/outlook/autolaunch.md [hunk 1]
```diff
@@ -29,8 +29,10 @@ We invite you to try out the event-based activation feature! Let us know your sc
...
-- [Configure targeted release on your Microsoft 365 tenant](/microsoft-365/admin/manage/release-options-in-office-365?view=o365-worldwide&preserve-view=true#set-up-the-release-option-in-the-admin-center).
+- For Outlook on the web:
+  - [Configure targeted release on your Microsoft 365 tenant](/microsoft-365/admin/manage/release-options-in-office-365?view=o365-worldwide&preserve-view=true#set-up-the-release-option-in-the-admin-center).
+  - Reference the **beta** library on the CDN (https://appsforoffice.microsoft.com/lib/beta/hosted/office.js). The [type definition file](https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts) for TypeScript compilation and IntelliSense is found at the CDN and [DefinitelyTyped](https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/types/office-js-preview/index.d.ts). You can install these types with `npm install --save-dev @types/office-js-preview`.
+- For Outlook on Windows: The minimum required build is 16.0.13729.20000. You may need to join the [Office Insider program](https://insider.office.com) for access to more recent Office builds.
```
Comment (JuaneloJuanelo) [inline docs/outlook/autolaunch.md:35]: They need to use the Beta Channel if using this feature before April 2021.

File: docs/outlook/autolaunch.md [hunk 2]
```diff
@@ -145,7 +148,7 @@ To enable event-based activation of your add-in, you must configure the [Runtime
 </VersionOverrides>
 ```
 
-Outlook on Windows uses a JavaScript file, while Outlook on the web uses an HTML file that references the same JavaScript file. You must provide references to both these files in the manifest as the Outlook platform ultimately determines whether to use HTML or JavaScript based on the Outlook client. As such, to configure event handling, provide the location of the HTML in the `Runtime` element, then in its `Override` child element provide the location of the JavaScript file inlined or referenced by the HTML.
+Outlook on Windows uses a JavaScript file, while Outlook on the web uses an HTML file that can reference the same JavaScript file. You must provide references to both these files in the manifest as the Outlook platform ultimately determines whether to use HTML or JavaScript based on the Outlook client. As such, to configure event handling, provide the location of the HTML in the `Runtime` element, then in its `Override` child element provide the location of the JavaScript file inlined or referenced by the HTML.
```
Comment (JuaneloJuanelo) [inline docs/outlook/autolaunch.md:151]: before this paragraph can we enumerate the 4  changes that are required on the file? a) adding version overrides 1_1 b) c) adding the Runtimes element, c) the LaunchEvents ExtensionPoint and d) the resources.

**Review by JuaneloJuanelo — CHANGES_REQUESTED** [review]
some minor comments added

File: docs/outlook/autolaunch.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/outlook/autolaunch.md:151]: @JuaneloJuanelo We discuss most of these at the top of the section. Is it necessary to repeat them?

**Pull request merged**